packages/components/src/custom-gradient-picker/gradient-bar/control-points.tsx

```diff
@@ -349,6 +350,7 @@ function InsertPoint( {
 			} }
 			renderToggle={ ( { isOpen, onToggle } ) => (
 				<Button
+					__next40pxDefaultSize
```

Comment:
The one other Button instance in this file is a `link` variant, so it doesn't need the prop.